### DISCUSSION TITLE
HomeKit Bridge lässt sich nicht mehr in Apple Home hinzufügen

### DISCUSSION THREAD

**Discussion by starauschek** [brutella/hkknx-public#284] [Q&A]
Hi,

nach einem Update der Apple Home App (iOS 16.7.2) lässt sich die Brücke nicht mehr als Gerät hinzufügen, wenn sie entfernt wurde. Es kommt immer die Fehlermeldung "Gerät bereits hinzugefügt" und "Gerät muss erst zurückgesetzt werden". Wie geht das? Kann mir jemand sagen, wie ich hkknx bzw. die HomeKit Bridge zurücksetzen kann?
Danke!

Grüße,
starauschek

**Comment by brutella** [comment]
In der Weboberfläche unter den Einstellungen → Mehr (Ganz Unten) → HomeKit Brücke zurücksetzen.

Ich habe dafür in der Online-Hilfe einen Eintrag erstellt. → https://hochgatterer.me/de/hkknx/docs/help/reset-accessory/

**Comment by starauschek** [comment]
Danke für den Hinweis und den Eintrag in der Online-Hilfe.

Jetzt habe ich ein anderes Problem, und zwar lässt mich Apple Home keine Personen mehr einladen. Beim Versuch bekomme ich die Meldung, dass ich eine Steuerzentrale benötige. Ich meine aber vorher ging das auch nur mit der Bridge. Mache ich hier (wieder) etwas falsch?

**Reply by brutella** [reply]
Seit der neuen Home-Architektur benötigst du zum einladen von Personen eine Steuerzentrale.

> Wenn dein Zuhause auf die [neue Home-Architektur](https://support.apple.com/HT213481) aktualisiert wurde, musst du [eine Steuerzentrale einrichten](https://support.apple.com/HT207057), um die Kontrolle über dein Zuhause zu teilen.
https://support.apple.com/de-de/HT208709
